clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/WgsCramKey.scala

```diff
@@ -1,15 +1,26 @@
 package org.broadinstitute.clio.transfer.model.wgscram
 
-import org.broadinstitute.clio.transfer.model.IndexKey
-import org.broadinstitute.clio.util.model.Location
-
-case class WgsCramKey(
-  location: Location,
-  project: String,
-  sampleAlias: String,
-  version: Int
-) extends IndexKey {
-
-  override def getUrlSegments: Seq[String] =
-    Seq(location.entryName, project, sampleAlias, version.toString)
+import org.broadinstitute.clio.util.model.{DataType, Location}
+
+//TODO delete when all other programs have transitioned to new cram API
+object WgsCramKey {
+
+  def apply(
+    location: Location,
+    project: String,
+    sampleAlias: String,
+    version: Int
+  ): WgsCramKey = new WgsCramKey(
+    location,
+    project,
+    sampleAlias,
+    version
+  )
 }
+
+class WgsCramKey(
```

Comment:
This doesn't need `case` to work with caseapp?

Reply:
nope